Cannot build offline version downloaded as a .zip

I downloaded master as a zip and after restoring all the packages from our internal repos, I'm unable to build. I think this maybe due to the fact that the .git directories are missing, meaning some kind of version discovery fails and stops the build.

Warnings like this:
```
"E:\csharp\dotnet-apiport-master\src\ApiPort\ApiPort\ApiPort.Offline.csproj" (default target) (1) ->
(WriteVersionInfoToBuildLog target) ->
  MSBUILD : warning : WARN [05/03/18 17:45:14:40] Could not determine assembly version: System.IO.DirectoryNotFoundException: Can't find the
.git directory in E:\csharp\dotnet-apiport-master\src\ApiPort\ApiPort [E:\csharp\dotnet-apiport-master\src\ApiPort\ApiPort\ApiPort.Offline.cs proj]
MSBUILD : warning :    at GitVersion.GitPreparer.GetDotGitDirectory() [E:\csharp\dotnet-apiport-master\src\ApiPort\ApiPort\ApiPort.Offline.cs proj]
MSBUILD : warning :    at GitVersion.ExecuteCore.ExecuteGitVersion(String targetUrl, String dynamicRepositoryLocation, Authentication authent ication, String targetBranch, Boolean noFetch, String workingDirectory, String commitId, Config overrideConfig, Boolean noCache) [E:\csharp\d otnet-apiport-master\src\ApiPort\ApiPort\ApiPort.Offline.csproj]
MSBUILD : warning :    at GitVersion.ExecuteCore.TryGetVersion(String directory, VersionVariables& versionVariables, Boolean noFetch, Authent ication authentication) [E:\csharp\dotnet-apiport-master\src\ApiPort\ApiPort\ApiPort.Offline.csproj]
```

Precede errors like this:
```
"E:\csharp\dotnet-apiport-master\src\ApiPort\ApiPort\ApiPort.Offline.csproj" (default target) (1) ->
"E:\csharp\dotnet-apiport-master\src\ApiPort\ApiPort\ApiPort.Offline.csproj" (Build target) (1:2) ->
"E:\csharp\dotnet-apiport-master\src\lib\Microsoft.Fx.Portability.MetadataReader\Microsoft.Fx.Portability.MetadataReader.csproj" (default tar get) (2:2) ->
"E:\csharp\dotnet-apiport-master\src\lib\Microsoft.Fx.Portability\Microsoft.Fx.Portability.csproj" (default target) (3:3) ->
(GetAssemblyVersion target) ->
  C:\Program Files\dotnet\sdk\2.1.4\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.GenerateAssemblyInfo.targets(146,5): error MSB4044: The "GetAs semblyVersion" task was not given a value for the required parameter "NuGetVersion". [E:\csharp\dotnet-apiport-master\src\lib\Microsoft.Fx.Po rtability\Microsoft.Fx.Portability.csproj]


"E:\csharp\dotnet-apiport-master\src\ApiPort\ApiPort\ApiPort.Offline.csproj" (default target) (1) ->
"E:\csharp\dotnet-apiport-master\src\ApiPort\ApiPort\ApiPort.Offline.csproj" (Build target) (1:3) ->
"E:\csharp\dotnet-apiport-master\src\lib\Microsoft.Fx.Portability.Reports.Html\Microsoft.Fx.Portability.Reports.Html.csproj" (default target)  (7:2) ->
"E:\csharp\dotnet-apiport-master\src\lib\Microsoft.Fx.Portability\Microsoft.Fx.Portability.csproj" (default target) (3:11) ->
  C:\Program Files\dotnet\sdk\2.1.4\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.GenerateAssemblyInfo.targets(146,5): error MSB4044: The "GetAs semblyVersion" task was not given a value for the required parameter "NuGetVersion". [E:\csharp\dotnet-apiport-master\src\lib\Microsoft.Fx.Po rtability\Microsoft.Fx.Portability.csproj]
```

--- a/Directory.Build.targets
+++ b/Directory.Build.targets
@@ -59,7 +59,7 @@
 
     <!-- GitVersionTask is not compiled against .NET Core, so importing the targets will
          result in a failure because it will try to resolve Microsoft.Build.Utilities v4.0 -->
-    <PackageReference Include="GitVersionTask" Version="4.0.0-beta0012" Condition=" '$(MSBuildRuntimeType)' != 'Core' ">
+    <PackageReference Include="GitVersionTask" Version="4.0.0-beta0012" Condition=" '$(MSBuildRuntimeType)' != 'Core' AND Exists('$(MSBuildThisFileDirectory)\.git') ">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
   </ItemGroup>

